codec-server/go.mod

```diff
@@ -0,0 +1,87 @@
+module github.com/temporalio/samples-go/codec-server
+
+go 1.21.1
+
+require (
+	github.com/golang/snappy v0.0.4
+	github.com/google/uuid v1.5.0
+	go.temporal.io/api v1.26.0
+	go.temporal.io/sdk v1.25.1
+	go.temporal.io/server v1.22.4
```

Comment:
I don't think the codec server sample should have to depend on the server library at all (other SDKs get along fine without it, though they don't have OIDC, but surely we can copy stuff). But I understand if now is not the time you want to remove this dependency.

Reply:
Yes I would rather do any changes like that in a separate PR